tdms/src/iterator.cpp

```diff
@@ -4930,16 +4881,13 @@ void mexFunction(int nlhs, mxArray *plhs[], int nrhs, const mxArray *prhs[])
     }//end of parallel section
     //fprintf(stderr,"Pos 09:\n");
     if(TIME_EXEC){
-      time_1=omp_get_wtime();
-      secs= time_1-time_0;
-      fprintf(stdout,"%.03e ",secs);
-      time_0=time_1;
+      timer.click();
     }
     /********************/
   
     //update terms for self consistency across scattered/total interface - E updates##
     if(sourcemode == sm_steadystate){//steadystate
-      commonPhase = exp(-I*fmod(omega_an[0]*time_H,2.*dcpi));
+      complex<double> commonPhase = exp(-I*fmod(omega_an[0]*time_H,2.*dcpi));

```

Comment:
I'm surprised you didn't use `auto` here :smile:

Reply:
auto _all_ the things 😉